tests/common/fixtures/duthost_utils.py

```diff
@@ -159,6 +162,70 @@ def ports_list(duthosts, rand_one_dut_hostname, rand_selected_dut, tbinfo):
     return ports
 
 
+def check_orch_cpu_utilization(dut, orch_cpu_threshold):
+    """
+    Compare orchagent CPU utilization 5 times, with 1 second interval in between and make sure all 5 readings are
+    less than threshold
+
+    Args:
+        dut: DUT host object
+        orch_cpu_threshold: orch cpu threshold
+    """
+    for i in range(5):
+        orch_cpu = dut.shell("COLUMNS=512 show processes cpu | grep orchagent | awk '{print $9}'")["stdout_lines"]
+        for line in orch_cpu:
+            if int(float(line)) > orch_cpu_threshold:
+               return False
+        time.sleep(1)
+    return True
+
+
+def check_ebgp_routes(num_v4_routes, num_v6_routes, duthost):
+    sumv4, sumv6 = duthost.get_ip_route_summary()
+    rtn_val = True
+    if 'ebgp' in sumv4 and 'routes' in sumv4['ebgp'] and sumv4['ebgp']['routes'] != num_v4_routes:
+        rtn_val = False
+    if 'ebgp' in sumv6 and 'routes' in sumv6['ebgp'] and sumv6['ebgp']['routes'] != num_v6_routes:
+        rtn_val = False
```

Comment:
i think the number of routes can vary before and after the test. Here [https://github.com/Azure/sonic-mgmt/blob/master/tests/platform_tests/link_flap/link_flap_utils.py#L229](https://github.com/Azure/sonic-mgmt/blob/master/tests/platform_tests/link_flap/link_flap_utils.py#L229) it takes a jitter of 5. Can you please check?

Reply:
Addressed in last commit